### DISCUSSION TITLE
Documentation on  `GET /api/getMetrics` endpoint

### DISCUSSION THREAD

**Discussion by SergeyVolynkin** [WalletConnect/walletconnect-monorepo#3686] [Developer Support]
Hi WalletConnect team,

We would want to import `Metrics` tab data available on the https://cloud.walletconnect.com/app into our internal data systems for processing and analysis.

Is there a way to periodically fetch `Metrics` data from WalletConnect currently?  Or all the data is only intended to be available via https://cloud.walletconnect.com/app user interface?

it seems that we would need to fetch `GET https://cloud.walletconnect.com/api/getMetrics?projectId=OUR_PROJECT_ID` and auth is via HTTP header Authorization (JTW token)

**Comment by glitch-txs** [comment]
Hi! we're working on refactoring the cloud website and we'll include Metrics data for each project.

**Reply by SergeyVolynkin** [reply]
Hi @glitch-txs, thank you for quick response

By `and we'll include Metrics data for each project.` do you mean that you'll provide some stable API endpoint to fetch the data for each project?

**Reply by glitch-txs** [reply]
Cloud projects will have a dashboard with the data available. Let me ask the team about the endpoint and I'll get back to you

**Reply by air-rick** [reply]
Thanks @glitch-txs. Having this exposed via an API endpoint would be great!

**Reply by GAEAlimited** [reply]
When will the refactoring of cloud website be completed?

**Comment by thisisrii** [comment]
Hi, WalletConnect team (@glitch-txs) 

I would also like to know if there's a way to extract this data either via API, or possible to add an export option in CSV / Excel as a first phase?  🙏
